RNSVG.podspec

```diff
@@ -3,15 +3,17 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name             = 'RNSVG'
```

Comment:
Why are all these lines marked as a diff?

Reply:
Because we're adding an additional space to account for `s.ios.exclude_files` and `s.osx.exclude_files`.